lib/SIL/DynamicCasts.cpp

```diff
@@ -1134,22 +1134,19 @@ bool swift::canUseScalarCheckedCastInstructions(SILModule &M,
     if (!objectType.isAnyClassReferenceType())
       return false;
     
-    if (M.getASTContext().LangOpts.EnableObjCInterop) {
       auto super = archetype->getSuperclass();
```

Comment:
The indentation here should be fixed to match the surrounding code.

Reply:
Ok!